src/people/people.repository.ts

```diff
@@ -11,6 +11,14 @@ const PEOPLE_ORDER_KEY_PATH_MAP: { [key in PeopleOrderKey]: string; } = {
     'name': 'name'
 };
 
+export interface NewPerson {
+    name: string;
+    about: string;
+    websiteUrl: string;
+    avatarUrl: string;
```

Comment:
Since we have avatar and artwork, it's essential to separate them into two folders in firestore.

Reply:
Oh! That's already implemented. Thx.